src/main/scala/com/fasterxml/jackson/module/scala/deser/TupleDeserializerModule.scala

```diff
@@ -5,7 +5,7 @@ import com.fasterxml.jackson.databind._
 import com.fasterxml.jackson.databind.deser.std.StdDeserializer
 import com.fasterxml.jackson.databind.deser.{BeanDeserializerFactory, Deserializers}
 import com.fasterxml.jackson.databind.jsontype.TypeDeserializer
-import com.fasterxml.jackson.module.scala.JacksonModule
```

Comment:
Just noticed this -- not wrong per se, but I wonder if there might be confusion now that Jackson 3.0 has renamed `Module` as `[com.fasterxml.jackson.databind.]JacksonModule`?

Reply:
It is a different class that does extend com.fasterxml.jackson.databind.JacksonModule (or Module). I'm open to renaming the Scala JacksonModule class but I'm not sure it is necessary.

Reply:
Right, understood. It is more an internal implementation detail it seems.
So naming is definitely up to you, I just thought I'd mention it.